gcloud/logging/logging_service_v2_api.py

```diff
@@ -38,47 +38,101 @@
 class LoggingServiceV2Api(object):
     """Service for ingesting and querying logs."""
 
-    # The default address of the logging service.
-    _SERVICE_ADDRESS = 'logging.googleapis.com'
+    _CODE_GEN_NAME_VERSION = 'gapic-0.1.0'
+
+    SERVICE_ADDRESS = 'logging.googleapis.com'
+    """The default address of the service."""
 
-    # The default port of the logging service.
-    _DEFAULT_SERVICE_PORT = 443
+    DEFAULT_SERVICE_PORT = 443
+    """The default port of the service."""
 
     # The scopes needed to make gRPC calls to all of the methods defined in
     # this service
-    _ALL_SCOPES = (
-        'https://www.googleapis.com/auth/logging.write',
-        'https://www.googleapis.com/auth/logging.admin',
-        'https://www.googleapis.com/auth/logging.read',
-        'https://www.googleapis.com/auth/cloud-platform.read-only',
-        'https://www.googleapis.com/auth/cloud-platform',
-    )
-
-    _LIST_LOG_ENTRIES_DESCRIPTOR = page_descriptor.PageDescriptor(
-        'page_token',
-        'next_page_token',
-        'entries',
-    )
-    _LIST_MONITORED_RESOURCE_DESCRIPTORS_DESCRIPTOR = page_descriptor.PageDescriptor(
-        'page_token',
-        'next_page_token',
-        'resource_descriptors')
-
-    def __init__(
-            self,
-            service_path=_SERVICE_ADDRESS,
-            port=_DEFAULT_SERVICE_PORT,
-            channel=None,
-            ssl_creds=None,
-            scopes=_ALL_SCOPES,
-            is_idempotent_retrying=True,
-            max_attempts=3,
-            timeout=30):
-        self.defaults = api_callable.ApiCallableDefaults(
-            timeout=timeout,
-            max_attempts=max_attempts,
-            is_idempotent_retrying=is_idempotent_retrying)
-        self.stub = api_utils.create_stub(
+    _ALL_SCOPES = ('https://www.googleapis.com/auth/logging.write',
+                   'https://www.googleapis.com/auth/logging.admin',
+                   'https://www.googleapis.com/auth/logging.read',
+                   'https://www.googleapis.com/auth/cloud-platform.read-only',
+                   'https://www.googleapis.com/auth/cloud-platform', )
```

Comment:
Sorry I didn't ask it up there but if `SERVICE_ADDRESS` and `DEFAULT_SERVICE_PORT` are public, why not make the scopes public? The value itself will end up in the docs for the constructor anyway (since it's the default).